EpiAware/src/initialisation.jl

```diff
@@ -1,30 +1,22 @@
-"""
-    default_initialisation_prior()
+abstract type AbstractInitialisation end
 
-Constructs a default initialisation prior for the model.
-
-# Returns
-`NamedTuple` with the following fields:
-- `I0_prior`: A standard normal distribution representing the prior for the initial infected population.
+struct SimpleInitialisation{D <: Sampleable, S <: Sampleable} <: AbstractInitialisation
+    mean_I0_prior::D
+    var_I0_prior::S
+end
 
-"""
 function default_initialisation_prior()
-    (; I0_prior = Normal(),)
+    (:mean_prior => Normal(), :var_prior => truncated(Normal(0.0, 0.05), 0.0, Inf)) |> Dict
```

Comment:
I've just deleted the whole initialisation file, that shouldn't have been committed.

I've downgraded the default initialisation prior functions in general to internal (not exported). In general, I'm leaning away from default priors because the context is so different in different model configurations.

For example, if we are modelling $r_t \sim RW(\sigma)$ then the initial value for the RW really shouldn't be wildly bigger than 0.23 (3 day doubling time if $t$ is days). Otoh if modelling $\log I_t \sim RW(\sigma)$ we might want a very different prior.

Since, this is composable having defaults feels a way of giving gotchas to a user.